platform/jewel/foundation/src/main/kotlin/org/jetbrains/jewel/foundation/JewelConfigs.kt

```diff
@@ -0,0 +1,7 @@
+package org.jetbrains.jewel.foundation
+
+@InternalJewelApi
+public object JewelConfigs {
```

Comment:
I'd call this either `JewelConfig` or, maybe better, `JewelFlags`. I also think we do not need to make it internal (users are still supposed to be able to use this, even if with caution). Then we want to annotate single flags as experimental (we may have non-experimental APIs over time).